ru_RU.json

```diff
@@ -0,0 +1,956 @@
+  "DEBUG_LOG_TREE_SCRIPT_EVENT": "Регистрация событий сценария",
+  "DEBUG_REMOVE_FROM_BAD_SPORT": "Удалить из Bad Sport",
+  "DEBUG_SCRIPT_EVENT_ARG": "Арг[%d]",
+  "DEBUG_SCRIPT_EVENT_ARG_COUNT": "Количество аргументов сценария:",
+  "DEBUG_SCRIPT_EVENT_EVERYONE": "Отправить всем",
+  "DEBUG_SCRIPT_EVENT_PLAYER_ID": "ID игрока:",
+  "DEBUG_SCRIPT_EVENT_SEND_EVENT": "Отправить событие",
+  "DEBUG_SCRIPTS_SUB_TITLE": "Эти скрипты отвечают за все зацикленные функции.\n",
+  "DEBUG_TAB_GLOBALS": "Глобальные",
+  "DEBUG_TAB_LOCALS": "Локальные",
+  "DEBUG_TAB_MISC": "Разное",
+  "DEBUG_TAB_SCRIPT_EVENTS": "События сценария",
+  "DEBUG_TAB_SCRIPTS": "Скрипты",
+  "DEBUG_TABS_LOGS": "Логи",
+  "DEBUG_WINDOW": "Отлаживать",
+  "DECLOAK": "Раскрыть",
+  "DEFAULT": "По умолчанию",
+  "DEFAULT_CMD_PERMISSIONS": "Разрешения команды по умолчанию",
+  "DELETE": "Удалить",
+  "DELETE_TRAIN": "Удалить поезд",
+  "DELETE_TRAIN_DESC": "Вы не можете удалить поезд, находясь в нем.",
+  "DERAIL_TRAIN": "Сойти с рельсов поезд",
+  "DESCENDING": "По убыванию",
+  "DESYNC_KICK": "Рассинхронный кик",
+  "DIRECTION": "Направление",
+  "DISABLE_CEO_MONEY": "Отключить деньги босса",
+  "DISABLE_CEO_MONEY_DESC": "Блокирует выпадение денег босса на протяжении всей сессии. ",
+  "DISABLE_COLLISION": "Отключить столкновение",
+  "DISABLE_ENGINE_AUTO_START": "Отключить автозапуск двигателя",
+  "DISABLE_FILTER": "Отключить фильтр",
+  "DISABLE_PEDS": "Отключить пешеходов",
+  "DISABLE_TRAFFIC": "Отключить трафик",
+  "DMG_MULTIPLR": "Множитель урона",
+  "DONT_STOP": "Не останавливаться",
+  "DRIVE_TRAIN": "Водить поездом",
+  "DRIVER": "Водитель",
+  "DRIVING_STYLE": "Стиль вождения",
+  "DRIVING_STYLE_SET_TO": "Стиль вождения установлен на {}.",
+  "DROWN": "Утопление",
+  "EMERGENCY_STOP": "Экстренная остановка",
+  "ENABLE_SPECIAL_AMMO": "Включить специальные боеприпасы",
+  "ENABLED": "Включено",
+  "END_KICK": "Кик завершения сеанса",
+  "END_KICK_DESC": "Это может занять около 10 секунд, чтобы кикнуть игрока.",
+  "ENTER_INTERIOR": "Войти в интерьер",
+  "ENTER_INTERIOR_DESC": "Входит внутрь игрока",
+  "ERROR_FAILED_TO_TAKE_CONTROL": "Не удалось взять под контроль транспортное средство игрока.",
+  "ERROR_INVALID_VEHICLE": "Подходящий автомобиль не найден.",
+  "ERROR_NO_OBJECTIVE": "Не удалось найти обьект",
+  "ERROR_NO_SEATS_FREE": "В автомобиле игрока нет свободных мест.",
+  "ERROR_NO_WAYPOINT_SET": "Не удалось найти положение путевой точки.",
+  "ERROR_PLAYER_IS_DEAD": "Целевой игрок мертв.",
+  "ERROR_PLAYER_IS_NOT_IN_VEHICLE": "Целевой игрок не находится в транспортном средстве.",
+  "ESP_BULLET": " ПУЛЯ",
+  "ESP_EXPLOSION": " ВЗРЫВ",
+  "ESP_GOD": "БОГ",
+  "EVENT_STARTER": "Стартер события",
+  "EXIT_TRAIN": "Выход из поезда",
+  "EXPLODE_ALL": "Взорвать всех",
+  "EXPLODE_PLAYER": "Взорвать игрока",
+  "EXPLODE_PLAYER_DESC": "Взрывает игрока",
+  "EXPLOSION": "Взрыв",
+  "EXPLOSION_KARMA": "Карма взрыва",
```

Comment:
I have no fucking idea what explosion karma means, and it's translated as such from the original, so I'm not gonna take points off from you, but @Yimura please explain what the hell this is.